core/test/integration/model/model_posts_spec.js

```diff
@@ -74,7 +74,7 @@ describe('Post Model', function () {
                 }).catch(done);
             });
 
-            it('can findAll, returning all related data', function (done) {
+            it.skip('can findAll, returning all related data', function (done) {
```

Comment:
ah haaaa! `findAll` tests can be removed, if you don't mind - that method was removed just the other day: https://github.com/TryGhost/Ghost/pull/5909/files